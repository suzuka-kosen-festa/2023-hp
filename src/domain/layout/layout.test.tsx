import { composeStory } from "@storybook/react";
import { render } from "@testing-library/react";
import Meta, { Default } from "./layout.stories";

describe("[domain/layout] layout.tsx", () => {
  it("should render the Footer component", () => {
    const Composed = composeStory(Default, Meta);
    const { container } = render(<Composed />);
    expect(container).toMatchSnapshot();
  });
});
