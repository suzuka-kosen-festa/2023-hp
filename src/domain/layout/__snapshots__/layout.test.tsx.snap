// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[domain/layout] layout.tsx > should render the Footer component 1`] = `
<div>
  <div
    class="styles_div__37tcib0"
  >
    <img
      alt="defect"
      class="styles_defect__37tcib1"
      src="/src/assets/icons/defect.svg"
    />
    <main
      class="styles_main__37tcib2"
    >
      Hello, world!
    </main>
    <footer
      class="styles_footer__1rq13180"
    >
      <p
        class="styles_p__1rq13181"
      >
        © 
        第58回鈴鹿高専祭 実行委員会
      </p>
    </footer>
  </div>
</div>
`;
